microsoft-edge/webview2/get-started/winui2.md

```diff
@@ -31,23 +27,29 @@ Follow the major Step sections in sequence, below.
 
 
 <!-- ====================================================================== -->
-## Step 1 - Download a preview channel of Microsoft Edge
+## Step 1 - Download the WebView2 Runtime
+<!-- todo: is this major step needed & correct? -->
```

Comment:
@jasonstephen15 - is this major step needed & correct?  "Download the WebView2 Runtime"
Rendered: https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/get-started/winui2?branch=pr-en-us-2043#step-1---download-the-webview2-runtime

Reply:
@jasonstephen15 - Removed section "Download the WebView2 Runtime".

Reply:
entered a non-Outdated fresh Comment, resolving this one